app/@search-bar/default.tsx

```
@@ -0,0 +1,46 @@
+"use client"
+
+import { ChangeEvent, useState } from "react";
+import { Input, Select } from "@/components/ui";
+
+type SelectOption = {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)